tests/testthat/test-api.R

```diff
@@ -0,0 +1,58 @@
+test_that("create_bugzilla_key works", {
+    vcr::use_cassette("create_bugzilla_key", {
+        cbk <- create_bugzilla_key(missing_host())
+        url <- "https://bugs.r-project.org/bugzilla/userprefs.cgi?tab=apikey"
+    })
+    expect_equal(paste0(missing_host(), "userprefs.cgi?tab=apikey"), url)
+    cli::test_that_cli("cli_alert_warning works", {
+        expect_snapshot({
+            cli::cli_alert_warning("This package hasn't been tested with this Bugzilla version.")
+        })
+    })
+})
+
+
+test_that("check_authentication works", {
+    vcr::use_cassette("check_authentication", {
+        ca <- check_authentication(missing_host())
+    })
+    expect_equal(missing_host(), "https://bugs.r-project.org/bugzilla/")
+    cli::test_that_cli("cli_alert_danger",{
+        expect_snapshot({
+            cli::cli_alert_danger("Not authenticated on this site.")
+            cli::cli_alert_success("Authenticated on this site!")
+        })
+    })
+})
+
+
+test_that("check_api_version works", {
+    vcr::use_cassette("check_api_version", {
+        cav <- check_api_version(missing_host())
+    })
+    expect_equal(missing_host(), "https://bugs.r-project.org/bugzilla/")
+})
+
+
+test_that("check_last_audit works", {
+    vcr::use_cassette("check_last_audit", {
+        cla <- check_last_audit(missing_product(), missing_host())
+    })
+    expect_equal(missing_host(), "https://bugs.r-project.org/bugzilla/")
+    expect_equal(missing_product(), "R")
+    expect_s3_class(GET(paste0(missing_host(), "rest/last_audit_time")), "response")
+})
+
+
+test_that("valid_key works", {
+    use_cassette("valid_key", {
+        vk <- valid_key(use_key())
+    })
+    cli::test_that_cli("cli_alert_dangers", {
+        expect_snapshot({
+            cli::cli_alert_danger("Key not valid or not set.")
+        })
+    })
+})
+
+snapshot_accept('R/api.R')
```

Comment:
I think, this call shouldn't be on the file as would make accept whatever this test produce. Snapshots should be created when the package is tested with `devtools::test()` or in RStudio. Please remove it. Also if you want to accept snapshots from a file it should have tests so it should point to "test-api.R" for example.